content/aws_marketplace.md

```diff
@@ -16,557 +16,114 @@ product = ["client", "workstation"]
-of tools and libraries that are packaged together to make it easy to
-develop cookbooks and resources for a Chef / Chef Automate environment.
-You'll need this to interact with Chef Automate and Chef Infra Server
-from the command line.
-
-## Configure Chef Automate
-
-After the instance has been provisioned and initial configuration has
-completed (usually 10 to 13 minutes) finish configuring Chef Automate
-and Chef Infra Server.
-
-1.  Access the initial configuration page by loading `/biscotti/setup`
-    route. Build the URL by prepending `https://` and appending
-    `/biscotti/setup` to the IP address or public hostname that was
-    automatically assigned to the instance when the Amazon Machine
-    Images (AMI) was launched. For example,
-    `https://<fqdn>/biscotti/setup`. If you used the BYOL image, the
-    CloudFormation stack will have the setup URL in the `Outputs`
-    section.
-
-    {{< note spaces=4 >}}
-
-    {{% notes_chef_aws_ssl %}}
+Chef Automate is an enterprise platform that allows developers, and operations, and security engineers to collaborate to deliver application and infrastructure changes at the speed and scale modern business requires. Chef Automate provides actionable insights across data centers and cloud providers where your nodes live.
 
-    {{< /note >}}
+Chef Automate is the center of the modern Chef platform, providing users with a single source of truth for infrastructure, security, and application automation. The comprehensive dashboard provides real-time views of your configuration management activity. Chef Automate comes bundled with the latest Chef Infra Server, providing the core tools you need to manage your enterprise infrastructure with Chef. Data collection is enabled by default, allowing your nodes to report activity in real-time. This instance is free for 60 days or you can bring your own license (BYOL).
 
-2.  Use the AWS console or command line tools to determine the Instance
-    ID of your Chef Automate instance. The instance ID is required for
-    authorization to access the setup page.
-
-3.  Fill out the setup form and submit it.
+Use this instance with Chef Workstation installed on your laptop or a separate AWS instance.
 
-4.  Follow the link and log into Chef Automate.
-
-## Configure the workstation
+{{< workstation >}}
 
-1.  Download and extract the `starter_kit.zip` file to a directory on
-    the workstation. Open a command prompt and change into the
-    `chef-repo` directory extracted from the starter kit. For example:
-
-    ```bash
-    cd ~/Downloads
-    unzip starter_kit.zip
-    cd starter_kit/chef-repo
-    ```
+In case you have a need of running automate in HA mode, please contact [Chef Support](https://www.chef.io/support).
 
-2.  {{% install_aws_chef_server_knife_client_list %}}
+## Installation
 
-## Configure backups
+TODO: Select the Chef Automate AWS Marketplace [CloudFormation template](NEED LINK
```

Comment:
https://aws-ami-chef-automate-v2.s3.amazonaws.com/cloudformation_template.yaml

Reply:
@kagarmoe : I have added the link. let me know. if any thing is needed apart from that